app/presenters/hyrax_show_presenter.rb

```diff
@@ -16,8 +16,44 @@ def member_presenter_factory
     ::EfficientMemberPresenterFactory.new(solr_document, current_ability, request)
   end
 
+  def export_as_jsonld
+    source_or_basic_jsonld.merge(local_fields).to_json
+  end
+
+  def export_as_nt
+    RDF::Graph.new.from_jsonld(export_as_jsonld).dump(:ntriples)
+  end
+
+  def export_as_ttl
+    RDF::Graph.new.from_jsonld(export_as_jsonld).dump(:ttl)
+  end
+
   private
 
+    def source_or_basic_jsonld
+      source_jsonld.nil? ? { title: title.first } : JSON.parse(source_jsonld)
+    end
+
+    def local_fields
+      {
+        '@context': 'https://bibdata.princeton.edu/context.json',
+        '@id': url,
+        memberOf: collection_objects,
+        scopeNote: portion_note,
+        navDate: nav_date,
+        edm_rights: solr_document.rights_statement,
+        identifier: identifier
+      }.reject { |_, v| v.nil? || v.try(:empty?) }
+    end
+
+    def url
+      Rails.application.routes.url_helpers.send "hyrax_#{model_name.singular}_url", id
+    end
+
+    def collection_objects
+      collection.map { |title| { title: title } }
```

Comment:
I hate to bring this up, but I wonder if this should include the @id for the collection too?